src/Azure/Orleans.Transactions.AzureStorage/Storage/AzureTransactionArchiveLogOptions.cs

```diff
@@ -2,7 +2,7 @@
 using System.Collections.Generic;
 using System.Text;
 
-namespace Orleans.Transactions.AzureStorage.Storage.Development
+namespace Orleans.Configuration.Development
```

Comment:
Is .Development the right namespace?

Is that because this is 'beta'/unstable or is it because it's not designed for production use?

Reply:
Ok, got it